pywinauto/unittests/test_uiawrapper.py

```diff
@@ -179,6 +180,45 @@ def test_is_dialog(self):
             button = self.dlg.child_window(class_name="Button",
                                            name="OK").wrapper_object()
             self.assertEqual(button.is_dialog(), False)
+            self.assertEqual(self.dlg.is_dialog(), True)
+
+        def test_move_window(self):
+            """Test  move_window without any parameters"""
+
+            # move_window with default parameters
+            prevRect = self.dlg.rectangle()
+            self.dlg.move_window()
+            self.assertEqual(prevRect, self.dlg.rectangle())
+
+            # move_window call for a not supported control
+            button = self.dlg.child_window(class_name="Button", name="OK")
+            self.assertRaises(NotImplementedError, button.move_window)
+
+            dlg_rect = self.dlg.parent().rectangle()  # use the parent as a reference
+            prev_rect = self.dlg.rectangle() - dlg_rect
+
+            new_rect = win32structures.RECT(prev_rect)
+            new_rect.left -= 1
+            new_rect.top -= 1
+            new_rect.right += 2
+            new_rect.bottom += 2
```

Comment:
Do we really need `win32structures` dependency in UIA tests? I believe it could look like 4-tuple: `(left, top, right, bottom) = prev_rect` but it produces `TypeError: 'RECT' object is not iterable` which should be fixed. `RECT` should be iterable with such a simple conversion to 4-tuple like we have for `POINT`.

Also I'm not sure that so commonly used `class RECT` is in a proper module. Maybe move it out from `win32structures.py`. But I have no idea for a new location yet. Maybe it's not needed right now.

Reply:
I added `RECT.__iter__` and opted out to named tuple in the test for now. Also to make linter happy changed the exception for `UIAWrapper.move_window` to AttributeError,